src/notebooks/notebookCreator.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { inject, injectable } from 'inversify';
```

Comment:
I believe this is a merge conflict problem. I deleted this file.

Reply:
This is what the command runs, so this file is necessary

Reply:
It was removed in the previous PR https://github.com/microsoft/vscode-jupyter/pull/9311/files

Reply:
Oh you removed it. HA! My bad.

Reply:
Not, sure why we're bringing this back? 
I thought the whole point of the previous PR was to remove this in favor of the VS code command
I